### PR TITLE
chore: Migrate/Distribute tests

### PR DESCRIPTION
**Pull request by satyamakgec**
# Description

# Integrations Checklist

- [ ] Have any function signatures changed? If yes, outline below.
- [ ] Have any features changed or been added? If yes, outline below.
- [ ] Have any events changed or been added? If yes, outline below.
- [ ] Has all documentation been updated?

# Changelog

### CODE REVIEW

**Review by lucas-manuel — DISMISSED** [review]
LGTM 👍

File: contracts/core/chainlink-oracle/v1/test/ChainLinkOracle.t.sol [hunk 1]
```diff
@@ -2,7 +2,7 @@
 pragma solidity 0.6.11;
 pragma experimental ABIEncoderV2;
 
-import "./TestUtil.sol";
+import "../../../../test/TestUtil.sol";
```
Comment (deluca-mike) [inline contracts/core/chainlink-oracle/v1/test/ChainLinkOracle.t.sol]: I'd prefer a remapping to a "global" `tests` directory, so we can avoid all of these ugly deep relative imports.

File: contracts/core/late-fee-calculator/v1/test/LateFeeCalc.t.sol [hunk 2]
```diff
@@ -0,0 +1,80 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+pragma solidity 0.6.11;
+pragma experimental ABIEncoderV2;
+
+import "../../../../test/TestUtil.sol";
```
Comment (deluca-mike) [inline contracts/core/late-fee-calculator/v1/test/LateFeeCalc.t.sol]: Ditto for this and all others.

File: contracts/core/pool/v1/test/PoolCustodial.t.sol [hunk 3]
```diff
@@ -2,9 +2,9 @@
 pragma solidity 0.6.11;
 pragma experimental ABIEncoderV2;
 
-import "./TestUtil.sol";
+import "../../../../test/TestUtil.sol";
 
-import "./user/Custodian.sol";
+import "../../../../test/user/Custodian.sol";
```
Comment (deluca-mike) [inline contracts/core/pool/v1/test/PoolCustodial.t.sol]: Ditto.

File: test.sh [hunk 4]
```diff
@@ -1,18 +1,20 @@
...
 set -e
 
-while getopts t:r:b: flag
+while getopts t:r:b:v: flag
 do
     case "${flag}" in
         t) test=${OPTARG};;
         r) runs=${OPTARG};;
         b) build=${OPTARG};;
+        v) version=${OPTARG};;
```
Comment (deluca-mike) [inline test.sh:10]: 💯

**Review by deluca-mike — CHANGES_REQUESTED** [review]

**Review by deluca-mike — APPROVED** [review]

**Pull request merged**